devscripts/prepare_manpage.py

```diff
@@ -65,8 +65,10 @@ def move_sections(readme):
 
 def filter_options(readme):
     section = re.search(r'(?sm)^# USAGE AND OPTIONS\n.+?(?=^# )', readme).group(0)
+    section_new = section.replace('*', r'\*')
```

Comment:
```suggestion
    section_new = section.replace('*', R'\*')
```
I think the uppercase in the original suggestion was to indicate a non-regex raw string